src/lib/actions/click-outside.ts

```diff
@@ -0,0 +1,16 @@
+/** Dispatch event on click outside of node */
+export function clickOutside(node: any) {
+  const handleClick = (event: MouseEvent) => {
+    if (node && !node.contains(event.target) && !event.defaultPrevented) {
+      node.dispatchEvent(new CustomEvent('clickOutside', node));
```

Comment:
I think the `CustomEventInit` is not required.

```suggestion
      node.dispatchEvent(new CustomEvent('clickOutside'));
```